packages/EasyCodingStandard/packages/ChangedFilesDetector/src/Cache/CacheFactory.php

```diff
@@ -20,9 +30,9 @@ private function getCacheDirectory(): string
     {
         if (defined('PHPUNIT_RUN')) { // defined in phpunit.xml
             // use different directory for tests, to avoid clearing local cache
-            return sys_get_temp_dir() . '/_changed_files_detector_tests';
+            return rtrim($this->cacheDirectory, '/') . '_tests';
```

Comment:
what if somebody will use it on other OS, 
```
DIRECTORY_SEPARATOR
```

Reply:
Good idea

Reply:
$cacheDirectory is provided by the user, not by the system. The default directory provided by EasyCodingStandardKernel is pretty safe. `DIRECTORY_SEPARATOR` will not solve it, if we want to be sure, we should check both backslach and slash.